ports/render96ex/render96ex.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+# PORTMASTER: sm64.zip, sm64.sh
+
+BASEROM='baserom.us.z64'
+RESTOOL_DIR="restool"
+RESTOOL_ZIP="restool.zip"
+
+# Ressources stuffs
+RES_DIR="res"
+BASEZIP="base.zip"
+DEMOS_DIR="demos"
+TEXTS_DIR="texts"
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+# We source the control.txt file contents here
+source $controlfolder/control.txt
+
+# With device_info we can get dynamic device information like resolution, cpu, cfw etc.
+source $controlfolder/device_info.txt
+
+# We source custom mod files from the portmaster folder example mod_jelos.txt which containts pipewire fixes
```

Comment:
Comments are nice for explanations how the .sh works but in the release version we usually keep them to a minimum. 
See https://portmaster.games/packaging.html#basic-launchscript for an example.